project/src/com/company/Main.java

```diff
@@ -0,0 +1,685 @@
+                    break;
+
+                case 10:
+                    card = chooseCard(scan, bank);
+                    if (card == null) break;
+
+                    bank.deleteCard(card);
+                    break;
+
+                case 11:
+                    System.out.print("The email of the account: ");
+                    email = scan.next();
+
+                    System.out.print("Amount added to savings: ");
+                    double savings_amount = scan.nextDouble();
+
+                    bank.addMoneyToSavings(email, savings_amount);
+                    break;
+
+                case 12:
+                    System.out.println("ACCOUNT WHICH PAYS THE AMOUNT:");
+                    Card card_paying = chooseCard(scan, bank);
+                    if (card_paying == null) break;
+
+                    System.out.println("ACCOUNT WHICH RECEIVES THE AMOUNT:");
+                    Card card_receiving = chooseCard(scan, bank);
+                    if (card_receiving == null) break;
+
+                    System.out.print("Amount payed: ");
+                    double amount = scan.nextDouble();
+
+                    if (!card_paying.canPay(amount)){
+                        System.out.println("Not enough funds to pay!");
+                        break;
+                    }
+
+                    bank.makeTransaction(card_paying, card_receiving, amount);
+                    break;
+
+                case 13:
+                    System.out.println("Bye bye! :)");
+                    return;
+                default:
+                    System.out.println("Option not available!%n");
+            }
+        }
+
+    }
+
+    private static Card chooseCard(Scanner scan, BankService bank){
+        System.out.print("The email of the account: ");
+        String email = scan.next();
+
+        Vector<Card> cards = bank.getCardsOfAccount(email);
+        if (cards == null || cards.size() == 0) {
+            System.out.println("Account currently has no cards!");
+            return null;
+        }
+
+        for (var item: cards) {
+            System.out.println(item.toString() + '\n');
+        }
+
+        int index_card = 0;
+        while (true){
+            System.out.print("Write the index of the card you want to use [0/1/2...]: ");
+            index_card = scan.nextInt();
+
+            if (index_card < 0 || index_card >= cards.size()){
+                System.out.print("Wrong index!");
+            }
+            else{
+                break;
+            }
+        }
+
+        return cards.get(index_card);
+    }
+}
+
+
+/*
+EXAMPLE:
+"C:\Program Files (x86)\jdk-17.0.2\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.3\lib\idea_rt.jar=54954:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.3\bin" -Dfile.encoding=UTF-8 -classpath "C:\Users\Alexandru Enache\IdeaProjects\project\out\production\project" com.company.Main
```

Comment:
is this example useful to have it in Main? you should keep this in README

Reply:
I have now created a .txt for the example in order to now add too many lines in README.